### ISSUE TITLE
Thanks!

### ISSUE THREAD

**Issue by nongdenchet** [manetivinay/Todo#1]
### Thank you for your submission
- Hi we meet again!!!
- First you really have a significant Android knowledge
- You really pushed effort on doing the prework, I really appreciate that
- We will send you an email soon to accept your submission
### Good stuffs
- Using `RecyclerView` instead of `ListView` is great
- Using `ContentProvider` for data sharing
- You have tweak wonderful UI here, very very beautiful really appreciate that
- Good use of `DialogFragment`, very beautiful timepicker and datepicker <3
- Good implementation of `SQLite`
- Nice to see `Parcelable` come in to action
- Good custom `ItemDecoration`
- Your naming convention is cool at all
- Overall your submission is awesome
### Suggestions
- Your features and UIs is awesome but the code is a little bit hard to follow. I do believe that we can work on this
### Questions
- How long have you developed Android? (just curious)

**Comment by manetivinay** [comment]
Hi Quan Vu, Nice to meet you. Thank you.I was trying to work most of the time on weekends . when ever I get some free time, I use to work. It took me around 3 - 4 days. I would like to concentrate more on Business logic and coding. 👍
